Dockerfile.api

```diff
@@ -8,7 +8,7 @@ RUN pip install --no-cache-dir --upgrade \
     uvicorn \
     joblib==1.2.0 \
     pandas==1.5.3 \
-    libpysal==4.7.0 \
+    libpysal==4.9.2 \
```

Comment:
The need for this update highlights the fact that the requirements are listed twice, once in pyproject.toml and once in the Dockerfile. This is a little annoying, and so far I haven't found a good way around it that doesn't involve (e.g.) manually parsing the pyproject.toml file, so I'll open a separate issue.